package/MDAnalysis/analysis/base.py

```diff
@@ -762,6 +769,10 @@ def run(
             Introduced ``backend``, ``n_workers``, ``n_parts`` and
             ``unsupported_backend`` keywords, and refactored the method logic to
             support parallelizable execution.
+
+        .. versionchanged:: 2.8.0
+            Set `n_parts` for parallel analysis to the number of frames
+            if `n_parts` is larger.
```

Comment:
Is this actually required? We haven't released 2.8.0 yet, so the `versionchanged` above is enough, I think?

Reply:
Yes, this can go.